content/tutorials/manage-the-rippled-server/configuration/configure-statsd.md

```diff
@@ -0,0 +1,57 @@
+# Configure StatsD
+
+ The client implementation of [StatsD](https://github.com/statsd/statsd) within rippled can be enabled to export health and behavioral information about the server. Those metrics can be consumed and visualized through [`rippledmon`](https://github.com/ripple/rippledmon) or any other collector that accepts StatsD formatted metrics. 
+
+## Configuration Steps
+
+ To enable StatsD on your rippled server, perform the following steps:
+
+1. Configure rippledmon on a separate machine. For more information about configuring `rippledmon` see the `rippledmon` [repository](https://github.com/ripple/rippledmon).
```

Comment:
Several small suggestions:

- Introduce the step with a little more context.
- Consistently use code font for `rippledmon`.
- Add a comma in "For more information about..., see"
- Extend the repository link to include the name.

All in all, here's my suggested revision:

> Set up a `rippledmon` instance on another machine to receive and aggregate stats. For more information about configuring `rippledmon`, see the [`rippledmon` repository](https://github.com/ripple/rippledmon).

It might also be handy to copy the basic console steps for setting up `rippledmon` here. My understanding is those are:

```sh
$ git clone https://github.com/rippled/rippledmon.git
$ cd rippledmon
$ docker-compose up
```